medusa/providers/generic_provider.py

```diff
@@ -659,13 +659,13 @@ def _create_anime_search_string(self, show_scene_name, episode, search_string, a
         """Create a search string used for as anime 'marked' shows."""
         episode_string = show_scene_name + self.search_separator
 
-        # If the show name is a season scene exception, we want to use the indexer episode number.
-        if (episode.scene_season > 0 and
-                show_scene_name in scene_exceptions.get_season_scene_exceptions(episode.series, episode.scene_season)):
```

Comment:
I'm not sure about this. It looks kinda like your removing this logic. It was added to prevent searches with a name like `show name season II` + absolute numbering. The check for season scene exception is gone now right?

Reply:
Why is it gone? I just replaced `episode.scene_season > 0` with `episode.series.is_scene` because it seemed a more explicit and correct way to make this check.